modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectRelationship.tsx

```diff
@@ -34,6 +34,7 @@ const headers = new Headers({
 
 const ModalAddObjectRelationship: React.FC<IProps> = ({
 	apiURL,
+	ffManyToManyRelationshipCustomAndSystemObjects,
```

Comment:
Hey @marcelabc, we can use the new way to put feature flags on the frontend: https://github.com/liferay-forms/liferay-portal/pull/1617. We can declare the Feature Flag LPS in `data-engine-js-components-web.d.ts` and use the `useFeatureFlag` hook return to make the conditions.